### DISCUSSION TITLE
Version v9.6-beta-1

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#473] [Announcements]
Version v9.6-beta-1

Fixed: _`showOriginalTextWhenHovering`_ always translates placeholder instead of value
: Update interface translations

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.6-beta-1'>Version v9.6-beta-1</a>.</em>

**Comment by RagihdHawanah** [comment]
The application is very important to me. I use the Brave browser, and it does not support translating pages in Arabic yet, so I used the extension 
it is very useful, but the problem is that after installing it a short period of time, errors appear and suddenly disappear and I have to reinstall it.
The problem is that I can not do without the addition, it is important to me

Is there any solution to this problem?
Problem quote:
Manifest version 2 is deprecated, and support will be removed in 2023. See https://developer.chrome.com/blog/mv2-transition/ for more details.

**Reply by RagihdHawanah** [reply]
and this problem als: 

Manifest version 2 is deprecated, and support will be removed in 2023. See https://developer.chrome.com/blog/mv2-transition/ for more details.
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
{
    "manifest_version": 2,
    "default_locale": "en",
    "name": "TWP - Translate Web Pages",
    "description": "__MSG_extensionDescription__",
    "version": "9.6.0.1",
    "homepage_url": "https://github.com/FilipePS/Traduzir-paginas-web",

    "commands": {
        "hotkey-toggle-translation": {
            "suggested_key": {
                "default": "Alt+T"
            },
            "description": "__MSG_lblSwitchTranslatedAndOriginal__"
        },
        "hotkey-translate-selected-text": {
            "suggested_key": {
                "default": "Alt+S"
            },
            "description": "__MSG_msgTranslateSelectedText__"
        },
        "hotkey-swap-page-translation-service": {
            "suggested_key": {
                "default": "Alt+Q"
            },
            "description": "__MSG_swapTranslationService__"
        },
        "hotkey-show-original": {
            "description": "__MSG_lblRestorePageToOriginal__"
        },
        "hotkey-translate-page-1": {
            "description": "__MSG_lblTranslatePageToTargetLanguage__ 1"
        },
        "hotkey-translate-page-2": {
            "description": "__MSG_lblTranslatePageToTargetLanguage__ 2"
        },
        "hotkey-translate-page-3": {
            "description": "__MSG_lblTranslatePageToTargetLanguage__ 3"
        }
    },

    "icons": {
        "32": "/icons/icon-32.png",
        "64": "/icons/icon-64.png",
        "128": "/icons/icon-128.png"
    },

    "permissions": [
        "<all_urls>",
        "storage",
        "activeTab",
        "contextMenus",
        "webRequest"
    ],

    "optional_permissions": [
        "webNavigation"
    ],

    "web_accessible_resources": [
        "/icons/icon-32.png",
        "/icons/google-translate-32.png",
        "/icons/yandex-translate-32.png",
        "/icons/swap-32.png",
        "/contentScript/css/*"
    ],

    "options_ui": {
        "page": "/options/options.html",
        "open_in_tab": false,
        "browser_style": false
    },

    "browser_action": {
        "browser_style": false,
        "default_icon": "/icons/icon-32.png",
        "default_title": "__MSG_pageActionTitle__"
    },

    "content_scripts": [{
            "matches": ["<all_urls>"],
            "run_at": "document_start",
            "all_frames": false,
            "js": ["/contentScript/checkScriptIsInjected.js"]
        },
        {
            "matches": ["<all_urls>"],
            "run_at": "document_start",
            "all_frames": true,
            "match_about_blank": true,
            "js": ["/lib/languages.js", "/lib/config.js", "/lib/platformInfo.js"]
        },
        {
            "matches": ["<all_urls>"],
            "run_at": "document_end",
            "all_frames": true,
            "match_about_blank": true,
            "js": ["/lib/i18n.js", "/contentScript/showOriginal.js", "/contentScript/pageTranslator.js", "/contentScript/translateSelected.js", "/contentScript/showTranslated.js"]
        },
        {
            "matches": ["<all_urls>"],
            "run_at": "document_end",
            "all_frames": false,
            "js": ["/contentScript/popupMobile.js"]
        },
        {
            "matches": ["https://www.deepl.com/translator*"],
            "run_at": "document_end",
            "all_frames": false,
            "js": ["/contentScript/deepl.js"]
        }
    ],

    "background": {
        "persistent": true,
        "scripts": ["/lib/languages.js", "/lib/config.js", "/lib/platformInfo.js", "/background/translationCache.js", "/background/translationService.js", "/background/textToSpeech.js", "/background/background.js"]
    }
}
 
TypeError: Failed to fetch
TypeError: Failed to fetch
TypeError: Failed to fetch
TypeError: Failed to fetch

**Reply by FilipePS** [reply]
Do not worry. As far as I know Brave will not remove support for manifest v2.
None of these messages appear to actually be an error in the extension.

**Reply by 505Games-oss** [reply]
I'm having the same problem as you today, I'm also using Brave Browser, and it's very likely that the junk cleaner we're using is mistakenly recognizing this plugin as the system cache. Please remove the checkbox of this plug-in when cleaning the system cache to solve the problem.

**Reply by fatihizm** [reply]
What you say is not the same as what happens. Brave Browser cannot bring this locally added add-on while syncing with the other computers.

**Comment by fatihizm** [comment]
I love this plugin, but since I use Brave, the plugin disappears when it syncs with the other computer. There must be a way to prevent this. For example, can you publish on the Google Chrome Web Store?

**Comment by RagihdHawanah** [comment]
its ok now the problem was solved
thanks for your interesting..

‫في الثلاثاء، 29 نوفمبر 2022 في 7:00 م تمت كتابة ما يلي بواسطة ‪Fatih‬‏ <‪
***@***.***‬‏>:‬
